fulp_modules/features/species/beefmen/beefman_clothing.dm

```diff
@@ -41,113 +41,113 @@
 
 /obj/item/clothing/under/bodysash/security/detective
 	name = "detective's sash"
-	desc = "A sash for someone that mean business."
+	desc = "A sash for someone who means business."
 	icon_state = "detective"
 
 /obj/item/clothing/under/bodysash/security/deputy
 	name = "deputy's sash"
-	desc = "An awe-inspiring \"tactical\" sash; because safety never takes a holiday."
+	desc = "An awe-inspiring \"tactical\" sash— because safety never takes a holiday."
 	icon_state = "deputy"
 
 //Medical
 /obj/item/clothing/under/bodysash/medical
 	name = "medical's sash"
-	desc = "A doctor's sash, It's made of a special fiber that provides minor protection against biohazards.."
+	desc = "A doctor's sash. It provides minor protection against biohazards."
 	icon_state = "medical"
 	armor_type = /datum/armor/clothing_under/rank_medical
 
 /obj/item/clothing/under/bodysash/medical/cmo
 	name = "chief medical officer's sash"
-	desc = "It's a sash worn by those with the experience to be \"Chief Medical Officer\". It provides minor biological protection."
+	desc = "A sash worn by those \"altruistic\" enough to become Chief Medical Officer. It provides minor biological protection."
```

Comment:
The quotations seem to be indicating irony (since nobody would put a common job title in quotation marks otherwise.) 

Moving the ironic quotations from the job title itself to the qualities commonly possessed by a person with the job maintains the same level of satirization while recognizing that the job is still an actual job.